src/qml/Mysthea/Pages/DetailDelegate.qml

```diff
@@ -20,51 +26,132 @@ Pane {
         ColumnLayout {
             id: _layout
             width: parent.width
-
-            spacing: 8
-
-            Label {
-                font.pixelSize: 24
-                text: root.code
-
-                Layout.alignment: Qt.AlignHCenter
-            }
-
-            RowLayout {
-
+            spacing: 24
+            ColumnLayout {
+                width: parent.width
```

Comment:
Since this `ColumnLayout` is inside another `Layout` you could use `Layout.fillWidth: true`.